database/factories/CharacterFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
+use App\Models\Character;
+
+class CharacterFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Character::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'user_id' => User::factory(),
+            'name' => $this->faker->name(),
+            'race' => $this->faker->word(),
+            'class' => $this->faker->word(),
+            'level' => $this->faker->word(),
```

Comment:
Was there a limit to the level a character can reach?

Reply:
There is! They can only be up to level 20, we can fix that.

Reply:
I didn't notice that the level was created as a word. Thanks for changing that too.